Dockerfile.autobuild

```diff
@@ -5,13 +5,34 @@
 # all of it's assets, there are more optimal ways to do this but this is the
 # most friendly and has everything contained in a single instance.
 #
-# Authors: Isaac Bythewood
-# Updated: Mar 29th, 2016
+# Authors: Isaac Bythewood, Jason Kaltsikis
+# Updated: May 2nd, 2020
 # Require: Docker (http://www.docker.io/)
 # -----------------------------------------------------------------------------
 
-# Base system is the LTS version of Ubuntu.
-FROM python:3.7-stretch
+# Build static yarn file
+FROM node:10.20.1-alpine3.11 as yarn-build
+
+WORKDIR pinry-spa
+COPY pinry-spa/package.json .
```

Comment:
Please copy the lock file `yarn.lock` to ensure the installtion install specified version of library.

Reply:
Done that, thank you.